nss-day-cohort-48/Rare-API-Django-Wandering-Wizards
# added a retrieve function in author view

#### Changes Made

1. Modified file `author.py` to include `retrieve` and functionality.

#### Steps to Review
1. Checkout to this branch locally
   ```
   git fetch --all
   git checkout ajw-retrieve-author
   ```

2. Ensure the backend and client are both running.

4. Test app functionality.
   
   > Example: Enter http://localhost:8000/authors/1 into postman and you should see details about a user
   > 

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
